pkgs/development/php-packages/grpc/default.nix

```
@@ -1,19 +1,28 @@
 { buildPecl
-, zlib
+, pkg-config
 , lib
+, grpc
 }:
 
 buildPecl {
   pname = "grpc";
-  version = "1.56.0";
+  version = grpc.version;
+  src = grpc.src;
 
-  sha256 = "sha256-uzxYMUzExMBDtwv3FipOuuUHg0v1wqAUtn69jXAQnf4=";
+  sourceRoot = "source/src/php/ext/grpc";
```

Slightly less fragile:

```suggestion
  sourceRoot = "${grpc.src.name}/src/php/ext/grpc";
```

Though ideally, we would have a separate variable for entering a directory after entering a source root.